docs/first-model/post-processing.rst

```diff
@@ -1,6 +1,36 @@
 Post processing
 ===============
 
-.. warning::
+Outputs
+-------
+**Fault (probe) outputs (2D)** 
 
-   This page is under construction.
++--------------------+------------------------------------------------------------------------+------+
+| Name               | Description                                                            | Unit |
++====================+========================================================================+======+
+| **Time**           | Time at which the output is written                                    | s    |
++--------------------+------------------------------------------------------------------------+------+
+| **state**          | State variable :math:`\phi = f_0 + b \ln{\left(V_0 \theta / L\right)}` |      |
```

Comment:
maybe remove the formula as it depends on the friction law?
(or maybe there is only one friction law implemented so far?)